pkgs/development/python-modules/langchain-community/default.nix

```diff
@@ -62,6 +64,10 @@ buildPythonPackage rec {
     "tenacity"
   ];
 
+  pythonRemoveDeps = [
+    "bs4"
```

Comment:
Why removing it? Is it not available?

Reply:
It's not available and is only used for formatting a single test's results.

Reply:
bs4 seems to be available... https://search.nixos.org/packages?channel=24.11&show=python312Packages.beautifulsoup4&from=0&size=50&sort=relevance&type=packages&query=beautifulsoup

Reply:
https://github.com/langchain-ai/langchain-community/issues/67